src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -0,0 +1,35 @@
+#ifndef DD_CLR_PROFILER_ENVIRONMENT_VARIABLES_H_
+#define DD_CLR_PROFILER_ENVIRONMENT_VARIABLES_H_
+
+#include "string.h"  // NOLINT
+
+namespace trace {
+namespace environment {
+
+const WSTRING tracing_enabled = "DD_TRACE_ENABLED"_W;
+
+const WSTRING debug_enabled = "DD_TRACE_DEBUG"_W;
+
+// supports multiple values
+const WSTRING integrations_path = "DD_INTEGRATIONS"_W;
+
+// supports multiple values
+const WSTRING process_names = "DD_PROFILER_PROCESSES"_W;
+
+const WSTRING agent_host = "DD_AGENT_HOST"_W;
+
+const WSTRING agent_port = "DD_TRACE_AGENT_PORT"_W;
+
+const WSTRING env = "DD_ENV"_W;
+
+const WSTRING service_name = "DD_SERVICE_NAME"_W;
+
+// supports multiple values
```

Comment:
Can you extend the comment with an example of that?

Reply:
Thanks. I [added comments](https://github.com/DataDog/dd-trace-dotnet/pull/237/commits/8f0626ca12aab939ba434cb9adb7fdc42d2eb8f5) to all the environment variables defined here.